### PR TITLE
docs(sdb): update faq and ref content MTA-4172

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: serverless/sql-databases/reference-content/serverless-sql-databases-overview.mdx [hunk 1]
```diff
@@ -38,10 +35,19 @@ Autoscaling triggers automatically based on your database vCPU consumption:
...
+
+  - **Scaling up**: after using more than 40% of the currently allocated vCPU for 10 seconds, the vCPU and RAM allocated to your database will increase by 1 vCPU and 4 GB RAM each time, up to a maximum of 15 vCPUs and 60 GB RAM.
+
+  - **Scaling down**: after using less than 2% of the currently allocated vCPU for 10 seconds, the vCPU and RAM allocated to your database will decrease by 1 vCPU and 4 GB RAM each time, down to a minimum of 1 vCPU and 4 GB RAM while your database is active.
+  
+<Message type="important">
+    These values may change over time to optimize the efficiency of automatic scaling operations.
+  </Message>
+
+Automatic scaling operations only happen **between** atomic transactions. For example, a long-running transaction (such as updating millions of rows) taking one minute will delay the next scaling operation by one minute. The next transaction performed will, however, benefit from the upscaled/downscaled database performance.
```
Comment (SamyOubouaziz) [inline serverless/sql-databases/reference-content/serverless-sql-databases-overview.mdx:50]: @fpagny This part is still "to assess"

File: faq/serverless-sql-databases.mdx [hunk 2]
```diff
@@ -50,5 +50,5 @@ You can avoid cold starts by provisioning a minimum of 1 vCPU or more.
 
 The following features are currently not supported and will be implemented at a later time:
 
-  - Automatic and manual backups
-  - Logs and metrics
\ No newline at end of file
+  - Manual backups (You can still perform manual backups using SQL queries or via SQL Administration tools such as PGAdmin)
```
Comment (RoRoJ) [inline faq/serverless-sql-databases.mdx:53]:
```suggestion
  - Manual backups (You can still perform manual backups using SQL queries or via SQL administration tools such as PGAdmin)
```

File: faq/serverless-sql-databases.mdx [hunk 3]
```diff
@@ -50,5 +50,5 @@ You can avoid cold starts by provisioning a minimum of 1 vCPU or more.
...
+  - Several SQL commands detailed in the [Known differences with default PostgreSQL](/serverless/sql-databases/reference-content/known-differences/)
```
Comment (RoRoJ) [inline faq/serverless-sql-databases.mdx:54]:
```suggestion
  - Several SQL commands detailed in the [Known differences with default PostgreSQL](/serverless/sql-databases/reference-content/known-differences/) documentation
```

File: serverless/sql-databases/concepts.mdx [hunk 4]
```diff
@@ -20,7 +20,7 @@ Autoscaling can automatically scale your database resources up or down, dependin
 
 ## Backup
 
-A database backup is a copy of the data stored in the database (including data structure, such as tables). Database backups in Serverless SQL Database are logical backups (as opposed to raw disk volume backups, also called "snapshots"). Database backups can be restored in the current database or any other PostgreSQL database and allows to recover from database crash, data corruption, or data loss.
+A database backup is a copy of the data stored in the database (including data structure, such as tables). Database backups in Serverless SQL Database are logical backups (as opposed to raw disk volume backups, also called "snapshots"). Database backups can be restored in the current database or any other PostgreSQL database and, allow to recover from database crash, data corruption, or data loss.
```
Comment (RoRoJ) [inline serverless/sql-databases/concepts.mdx:23]:
```suggestion
A database backup is a copy of the data stored in the database (including data structure, such as tables). Database backups in Serverless SQL Database are logical backups (as opposed to raw disk volume backups, also called "snapshots"). Database backups can be restored in the current database or any other PostgreSQL database and, allow recovery from database crashes, data corruption, or data loss.
```

File: serverless/sql-databases/troubleshooting/failing-backup-restore.mdx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  posted: 2024-02-07
+categories:
+  - serverless
+---
+
+## Manual SQL backup and restore fails
+
+### Problem
+
+When creating or restoring backup manually (i.e. via SQL Administration tools such as PGAdmin, or directly via SQL commands), some errors can happen.
```
Comment (RoRoJ) [inline serverless/sql-databases/troubleshooting/failing-backup-restore.mdx:20]: The capitalization of PGAdmin is not always consistent between the different docs

Comment (RoRoJ) [inline serverless/sql-databases/troubleshooting/failing-backup-restore.mdx:20]:
```suggestion
When creating or restoring backup manually (i.e. via SQL Administration tools such as PGAdmin, or directly via SQL commands), some errors can occur.
```

File: serverless/sql-databases/troubleshooting/connection-errors.mdx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+| Python   | psycopg2              | Yes           |                                                                                 |
+| Python   | SQLAlchemy            | Yes           |                                                                                 |
+| Java     | pgjdbc                | Yes           |                                                                                 |
+| Go       | lib/pq                | Partial       | Upgrade lib/pq to v1.10.7, which supports SNI.                                  |
+| Dart     | postgres              | Yes           |                                                                                 |
+| N/A      | psql                  | Yes           |                                                                                 |
+| N/A      | pgcli                 | Yes           |                                                                                 |
+| N/A      | PGAdmin               | Yes           |                                                                                 |
```
Comment (SamyOubouaziz) [inline serverless/sql-databases/troubleshooting/connection-errors.mdx:63]:
```suggestion
| N/A      | pgAdmin               | Yes           |                                                                                 |
```

File: serverless/sql-databases/troubleshooting/failing-backup-restore.mdx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  posted: 2024-02-07
+categories:
+  - serverless
+---
+
+## Manual SQL backup and restore fails
+
+### Problem
+
+When creating or restoring backup manually (i.e. via SQL Administration tools such as PGAdmin, or directly via SQL commands), some errors can occur.
```
Comment (SamyOubouaziz) [inline serverless/sql-databases/troubleshooting/failing-backup-restore.mdx:20]:
```suggestion
When creating or restoring backup manually (i.e. via SQL Administration tools such as pgAdmin, or directly via SQL commands), some errors can occur.
```

File: serverless/sql-databases/troubleshooting/failing-backup-restore.mdx [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+### Cause
+
+These issues are caused by using `pg_dump` and `pg_restore` versions that are not compatible with PostgreSQL 16 (on which Serverless SQL Databases rely).
+
+### Solution
+
+To solve these issues, upgrade your `pg_dump` and/or `pg_restore` modules:
+
+- You can upgrade them by installing PostgreSQL 16 which includes these tools.
+- If you are using a third-party tool that includes these libraries, upgrade your tool. For instance, PGAdmin supports PostgreSQL 16 ecosystem from version 7.8.
```
Comment (SamyOubouaziz) [inline serverless/sql-databases/troubleshooting/failing-backup-restore.mdx:31]:
```suggestion
- If you are using a third-party tool that includes these libraries, upgrade your tool. For instance, pgAdmin supports PostgreSQL 16 ecosystem from version 7.8.
```

**Review by RoRoJ — APPROVED** [review]

File: serverless/sql-databases/reference-content/serverless-sql-databases-overview.mdx (hunk 1 above)
Comment (fpagny) [reply serverless/sql-databases/reference-content/serverless-sql-databases-overview.mdx]: No it has been cleared with engineering, although it's not deployed into production yet. This part will be true only in March when deployment is done.

File: faq/serverless-sql-databases.mdx [hunk 9]
```diff
@@ -50,5 +50,5 @@ You can avoid cold starts by provisioning a minimum of 1 vCPU or more.
 
 The following features are currently not supported and will be implemented at a later time:
 
-  - Automatic and manual backups
-  - Logs and metrics
\ No newline at end of file
+  - Manual backups (You can still perform manual backups using SQL queries or via SQL administration tools such as pgAdmin)
```
Comment (nerda-codes) [inline faq/serverless-sql-databases.mdx:53]:
```suggestion
  - Manual backups (you can still perform manual backups using SQL queries or via SQL administration tools such as pgAdmin)
```

File: serverless/sql-databases/troubleshooting/connection-errors.mdx [hunk 10]
```diff
@@ -0,0 +1,69 @@
+---
+meta:
+  title: Solving connection errors
+  description: This page lists the different solutions to solve connection errors with Serverless SQL Databases.
```
Comment (nerda-codes) [inline serverless/sql-databases/troubleshooting/connection-errors.mdx:4]:
```suggestion
  description: This page lists different solutions to connection errors with Serverless SQL Databases.
```

File: serverless/sql-databases/troubleshooting/connection-errors.mdx [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+content:
+  h1: Solving connection errors
+  paragraph: This page lists the different solutions to solve connection errors with Serverless SQL Databases.
```
Comment (nerda-codes) [inline serverless/sql-databases/troubleshooting/connection-errors.mdx:7]:
```suggestion
  paragraph: This page lists different solutions to connection errors with Serverless SQL Databases.
```

File: serverless/sql-databases/troubleshooting/failing-backup-restore.mdx [hunk 12]
```diff
@@ -0,0 +1,31 @@
+---
+meta:
+  title: Solving failing manual backup and restore operations
+  description: This page lists the different solutions to solve failing backup and restore operations with Serverless SQL Databases.
```
Comment (nerda-codes) [inline serverless/sql-databases/troubleshooting/failing-backup-restore.mdx:4]:
```suggestion
  description: This page lists different solutions to failing backup and restore operations with Serverless SQL Databases.
```

File: serverless/sql-databases/troubleshooting/failing-backup-restore.mdx [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+content:
+  h1: Solving failing manual backup and restore operations
+  paragraph: This page lists the different solutions to solve failing backup and restore operations with Serverless SQL Databases.
```
Comment (nerda-codes) [inline serverless/sql-databases/troubleshooting/failing-backup-restore.mdx:7]:
```suggestion
  paragraph: This page lists different solutions to failing backup and restore operations with Serverless SQL Databases.
```

File: serverless/sql-databases/troubleshooting/failing-backup-restore.mdx [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+  posted: 2024-02-07
+categories:
+  - serverless
+---
+
+## Manual SQL backup and restore fails
+
+### Problem
+
+When creating or restoring backup manually (i.e. via SQL Administration tools such as pgAdmin, or directly via SQL commands), some errors can occur.
```
Comment (nerda-codes) [inline serverless/sql-databases/troubleshooting/failing-backup-restore.mdx:20]:
```suggestion
When creating or restoring backup manually (i.e. via SQL Administration tools such as pgAdmin, or directly via SQL commands), some errors may occur.
```

File: serverless/sql-databases/concepts.mdx [hunk 15]
```diff
@@ -20,7 +20,7 @@ Autoscaling can automatically scale your database resources up or down, dependin
 
 ## Backup
 
-A database backup is a copy of the data stored in the database (including data structure, such as tables). Database backups in Serverless SQL Database are logical backups (as opposed to raw disk volume backups, also called "snapshots"). Database backups can be restored in the current database or any other PostgreSQL database and allows to recover from database crash, data corruption, or data loss.
+A database backup is a copy of the data stored in the database (including data structure, such as tables). Database backups in Serverless SQL Database are logical backups (as opposed to raw disk volume backups, also called "snapshots"). Database backups can be restored in the current database or any other PostgreSQL database and, allow recovery from database crashes, data corruption, or data loss.
```
Comment (nerda-codes) [inline serverless/sql-databases/concepts.mdx:23]: Just wondering about the wording of "Serverless SQL Database" (if the "d" should be capitalized or not)

Comment (SamyOubouaziz) [inline serverless/sql-databases/concepts.mdx:23]:
```suggestion
A database backup is a copy of the data stored in the database (including data structure, such as tables). Database backups in Serverless SQL Databases are logical backups (as opposed to raw disk volume backups, also called "snapshots"). Database backups can be restored in the current database or any other PostgreSQL database and, allow recovery from database crashes, data corruption, or data loss.
```

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**